### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by NikVolf**
after #17 is decided

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,38 +55,29 @@ sudo apt-get install build-essential
...
+cd parity-zcash
 
-```
 # builds pbtc in release mode
-cargo build -p pbtc --release
+cargo build -p pzec --release
 ```
 
-`pbtc` is now available at either `./target/debug/pbtc` or `./target/release/pbtc`.
+`pzec` is now available at either `./target/release/pbtc`.
```
Comment (svyatonik) [inline README.md]: `./target/release/pzec` ?

Comment (svyatonik) [reply README.md]: + redundant 'either'

File: README.md [hunk 2]
```diff
@@ -99,84 +87,54 @@ Assuming that repository is already cloned, we can run unit tests with this comm
...
-By default parity connects to bitcoind-seednodes. Full list is [here](./pbtc/seednodes.rs).
-
-Before starting synchronization, you must decide - which fork to follow - Bitcoin Core (`--btc` flag) or Bitcoin Cash (`--bch` flag). On next start, passing the same flag is optional, as the database is already bound to selected fork and won't be synchronized using other verification rules.
+By default parity connects to Zcash seednodes. Full list is [here](./pzec/seednodes.rs).
 
 To start syncing the main network, just start the client, passing selected fork flag. For example:
 
 ```
-./target/release/pbtc --btc
+./target/release/pzec --btc
```
Comment (svyatonik) [inline README.md]: obsolete `--btc` flag?

**Comment by NikVolf** [comment]
> need some love

my thoughts exactly)

**Pull request merged**